config/SupportedVehicles.xml

```diff
@@ -96,6 +96,30 @@
         <vehicleConfiguration configFileName="FS22_Lizard_Selfmade_Tow_Truck/pickup2017.xml">
             <loadingArea offset="0 0.65 -2.70" width="2.0" height="2.20" length="4.25"/>
             <options noLoadingIfUnfolded="true"/>
+		</vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_allPurposeTool/all-purpose-tool.xml">
+            <loadingArea offset="0 0.928 0.819" width="3.436" height="1.602" length="1.704"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPX_816/spx816.xml">
+            <loadingArea offset="0.028 2.629 1.348" width="2.195" height="2.339" length="7.78"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/RPX_920/rpx920.xml">
+            <loadingArea offset="0.028 2.629 0.791" width="2.195" height="2.339" length="8.662"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/RPX_1226/rpx1226.xml">
+            <loadingArea offset="0.028 2.703 -0.657" width="2.195" height="2.339" length="11.831"/>
+        </vehicleConfiguration>
+        <vehicleConfiguration configFileName="FS22_PongePack/SPEX_7519/SPEX.xml">
```

Comment:
Looks pretty good. Could always increase the width and length a little, but no major complaints. Height offset is too high above the bed like the others. Also, could add ```<options enableRearLoading="true"/>``` if we want to enable loading via drive up the ramps with forklifts. No convinced this trailer needs it though.